packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -66,17 +69,9 @@
     opacity: $bpk-modal-opacity;
   }
 
-  &--iphone-fix {
```

Comment:
FYI there are still instances of `--iphone-fix` in the `withScrim` HOC (see `bpk-scrim-utils`). 

It'd be worth revisiting these...

Reply:
I think we can remove that too. It doesn't seem to change anything